test/src/tests/generated/lro/models.go

```diff
@@ -1549,6 +1549,64 @@ func (future *LROsPostAsyncRetrySucceededFuture) Result(client LROsClient) (p Pr
 	return
 }
 
+// LROsPostDoubleHeadersFinalAzureHeaderGetFuture an abstraction for monitoring and retrieving the results of a
+// long-running operation.
+type LROsPostDoubleHeadersFinalAzureHeaderGetFuture struct {
```

Comment:
>LROsPostDoubleHeadersFinalAzureHeaderGetFuture [](start = 5, length = 46)

are these coming from new tests in the test server ?

Reply:
in that case is there any extra test we need to add ?

---
In reply to: [196148727](https://github.com/Azure/autorest.go/pull/134#discussion_r196148727) [](ancestors = 196148727)